guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -22,27 +20,46 @@ ifdef::foreman-el,katello,satellite[]
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
+
+|/var/log |10 MB |10 GB
+
+|/var/lib/pgsql |100 MB |20 GB
+
+|/usr | 5 GB | Not Applicable
+
+|/opt/puppetlabs | 500 MB | Not Applicable
+
 ifdef::katello,satellite,orcharhino[]
 |/var/lib/pulp |1 MB |300 GB
+
+|/var/lib/qpidd |25 MB | Not Applicable
```

Comment:
the "Not Applicable" is not entirely correct, as later in `guides/common/modules/ref_storage-guidelines.adoc` we do give guidance how this grows when you add content hosts with katello-agent installed

Reply:
Okk. Shall I go for `20 GB` instead of `Not Applicable` ?

Reply:
Can you put a reference to the chapter where it's discussed? 20G is too much for *most* users (especially as this only happens when you use the deprecated Katello agent)

Reply:
I read [here](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html-single/installing_satellite_server_in_a_connected_network_environment/index#storage-guidelines_satellite). If this is correct, we can replace this link with `Not Applicable`.

Reply:
Hi @evgeni 
There is also a suggestion of `50 MB` from [here](https://bugzilla.redhat.com/show_bug.cgi?id=2011500)

Reply:
Which doesn't make sense whatsoever (honestly, both, 20 and 50MB are nonsense values).

In the guidelines we write "The `/var/lib/qpidd/` directory uses slightly more than 2 MB per Content Host managed by the `goferd` service.", so the runtime size should either refer to that statement, or duplicate it.

Reply:
Or we could keep this PR about the original issue, `/usr`, and discuss qpidd later.

Reply:
If we resolve `qpidd` here, it will be an addon for me. However, I am fine with either one.

Reply:
> Which doesn't make sense whatsoever (honestly, both, 20 and 50MB are nonsense values).
> 
> In the guidelines we write "The `/var/lib/qpidd/` directory uses slightly more than 2 MB per Content Host managed by the `goferd` service.", so the runtime size should either refer to that statement, or duplicate it.

Added internal link to Storage Guidelines in the table. :+1: